pkg/apis/pytorch/v1/types.go

```diff
@@ -69,6 +69,10 @@ type PyTorchJobSpec struct {
 	//     "Worker": PyTorchReplicaSpec,
 	//   }
 	PyTorchReplicaSpecs map[PyTorchReplicaType]*common.ReplicaSpec `json:"pytorchReplicaSpecs"`
+
+	//添加判断优先级的属性
+	//add PriorityClassName
+	PriorityClassName string `json:"priorityClassName,omitempty"`
```

Comment:
```suggestion
	PriorityClassName *string `json:"priorityClassName,omitempty"`
```

Since it is optional, we can define it as a pointer.